### DISCUSSION TITLE
Anybody have a guess which plugin is messing up indent repetition?

### DISCUSSION THREAD

**Discussion by llimllib** [LazyVim/LazyVim#1239] [Q&A] [answered]
In a buffer, visually select two lines of text, indent them and then try to repeat: `Vj>.....`

In a plain nvim, this will indent the two lines as many times as you repeat. In LazyVim, it will only indent them once.

I'd really like for it to indent the lines as many times as I repeat, but I can't figure out which plugin is breaking it.

I've tried disabling some plugins that I thought might be at fault, but no luck. Given there's a lot of plugins in play here, anybody have a guess, or a good strategy for trying to figure out what plugin is at fault?

**Comment by folke** [comment]
you can repeat with `>` again

**Reply by llimllib** [reply]
I mean, yes, but I want to repeat with `.`

**Reply by llimllib** [reply]
(I'm not blaming you or anybody else, but this is hard-wired into my fingers from many years and I'm willing to disable pretty much whatever plugin no matter how nice to get my editing to work like my fingers expect)

**Reply by folke** [reply, marked as answer]
then you'll have to unmap `<` and `>`

**Reply by llimllib** [reply]
Took me a minute to figure out how to unmap, you actually just set it to the default mapping I guess:

```
vim.keymap.set("v", "<", "<")
vim.keymap.set("v", ">", ">")
```

did the trick. Thanks!

The culprit was [these mappings](https://github.com/LazyVim/LazyVim/blob/142e6be/lua/lazyvim/config/keymaps.lua#L90-L92)

**Reply by llimllib** [reply]
One last question: in what way is that mapping supposed to impove indenting?

**Reply by folke** [reply]
because I like it that way? if you don't then remove the mapping

**Reply by llimllib** [reply]
apologies, I just literally don't understand what it does, not suggesting it's bad. Was asking out of curiosity

**Reply by folke** [reply]
you can repeat the indent with `>` and `<`
